packages/discovery-react-components/src/components/DocumentPreview/components/Highlight/__tests__/passage.test.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { FC } from 'react';
+import docJson from 'components/DocumentPreview/__fixtures__/Art Effects Koya Creative Base TSA 2008.pdf.json';
+import { usePassage, isPassage } from '../passages';
+import { QueryResultPassage } from 'ibm-watson/discovery/v2';
+import { act, render } from '@testing-library/react';
+
+describe('Passage', () => {
+  it('returns passage bbox data from document', () => {
+    const passage = {
+      passage_text: '5.21 Miscellaneous Costs',
+      start_offset: 39611,
+      end_offset: 39635,
+      field: 'text'
+    };
+
+    let bboxes;
+
+    // Since usePassage uses hooks got to wrap in a test component to test it
+    const TestComponent: FC = () => {
+      bboxes = usePassage(docJson, passage as QueryResultPassage);
+      return <div>Test</div>;
+    };
```

Comment:
Not sure about this, with the "state" living outside of our test component. I know it works, but I'm wondering on what the best practices are for testing hooks.

Reply:
Ooh, take a look at https://github.com/testing-library/react-hooks-testing-library